modules/fedimint-ln-client/src/lib.rs

```diff
@@ -125,6 +154,24 @@ pub enum LnReceiveState {
     Claimed,
 }
 
+async fn check_internal_payment(
```

Comment:
I think we should check whether the route hint matches **any** gateway, not just the active one.

Reply:
ack